modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/FunctionNode.java

```diff
@@ -47,14 +63,25 @@ public BlockNode getBlockNode() {
 
     /* ---- end tree structure, begin node data ---- */
 
+    private ScriptRoot scriptRoot;
     private String name;
     private Class<?> returnType;
     private List<Class<?>> typeParameters = new ArrayList<>();
     private List<String> parameterNames = new ArrayList<>();
+    private boolean isStatic;
     private boolean isSynthetic;
+    private boolean doAutoReturn;
```

Comment:
Even with the name change, a comment on the semantics of this var would be useful since it's a non-standard extension.

Reply:
Done.